packages/components/tooltip/src/Tooltip.tsx

```diff
@@ -91,23 +90,24 @@ export interface TooltipProps {
 export const Tooltip = ({
   children,
   className,
-  containerElement: ContainerElement = 'span',
+  as: HtmlTag = 'span',
   content,
-  id,
+  id = 'cf-ui-tooltip',
```

Comment:
this id should be optional and generate a unique number if the value is not passed

Reply:
use could use a convenient hook from Charka: https://github.com/chakra-ui/chakra-ui/blob/3205ad6d55f935f37db8f61d2dde7eaa2481578c/packages/tooltip/src/use-tooltip.ts#L108